### ISSUE TITLE
Add "Definitions"?

### ISSUE THREAD

**Issue by erbenpeter** [xandrew/agree2disagree#39]
My personal experience is that one typical difficulty in hard to resolve debates is that we have different definitions of the words / concepts we use. Typical contemporary examples: "freedom", "harm", "woman". Sometimes it's "organic" in the sense that different cultures have different understanding of basic concepts (such as "family" or "dignity") and sometimes it's the consequence of deliberate political actions: the actors want to shift the meaning of words to achieve a political goal.

IMO this is a fourth item for your list on the help page (different values, different information, deliberate mischief). Similar to values, but one level deeper. For example the value "It's ok to sometimes cause small harm for the greater good." depends on the definition of "harm". 

My idea is to add a feature which separates out this problem, making it explicit that during the debate we used different definitions for the same words. Some way to mark single words and open a definition thread. It's different from the main argument thread. Maybe the "disagreers" can mark words which need definition and everyone can contribute their own definition without arguing about these. 

The difficulty is that it could easily become circular. Which brings us to the concept of axioms which can be used to fill the base concepts with "meaning". :)

**Comment by xandrew** [comment]
Indeed, this is an interesting and valid point. What seems to be difficult though is that I'm not sure it's OK to leave definitions "undefined" the same way it's OK to leave claims undecided. I mean, if we understand the meaning of a claim differently than the argument is doomed to achieve nothing.

So I think we do need to settle on a single definition, at least in the context of one given text.

This then evokes the question of who is the one to decide, which in my mind is more or less the same problem as the ones discussed in #33.

I'm not sure I fully understand your point about the axioms, though. Don't they come somehow _after_ the definitions, as claims accepted without proof about defined stuff?

**Comment by erbenpeter** [comment]
> I'm not sure I fully understand your point about the axioms, though. 

I just wanted to refer to the mathematical problem that for defining the meaning of words we need to write down our definitions using other _words_. One mathematical way to avoid an infinite loop is to accept that some concepts cannot be defined (I think _base concepts_ is the phrase). To use these base concepts / words in a debate we need to have axioms which describe how these _things_ behave and ideally we only use these axiomatically defined properties of our things when reasoning in the debate.

I agree with your point that questioned definitions need to be settled somehow to have a meaningful discussion later.
But as I see it's already an achievement if we can realize, that our definitions didn't match originally.

In a non-hostile environment my approach would be to "split" the debate into separate threads by the different definitions. Sometimes it turns out that we can come to the same conclusion using different definitions because we didn't have to use different axioms / properties. In a hostile environment it's easy to stuck at this first step if we insist on using non-compatible definitions. But my impression is that `agrre2disagree` is designed to create non-hostile environments for debates, this is why I thought that leaving a definition unsettled can be a more practical approach than requiring  the users to use axiomatic reasoning. In that case the marker is just a reminder, that there is a possibility that our definitions differ.
